package.json

```diff
@@ -12,11 +12,12 @@
         "test-watch": "jest --watch"
     },
     "dependencies": {
+        "@quasar/cli": "^1.2.0",
```

Comment:
Quasar CLI probably only needs to be in devDepencies?

Reply:
I don't think it needs to be a dependency at all. I accidentally added it when trying to update it (was supposed to be a global update instead of local).